schoolopy/main.py

```diff
@@ -129,22 +143,90 @@ def create_building(self, school_id, building):
         """
         return Building(self._post('schools/%s/buildings' % school_id, building.json()))
 
-    def get_users(self):
+    def get_me(self):
+        """
+        Get data of the client using this method (Yourself)
+
+        :return: User object obtained from API. (Of yourself)
+        """
+        return User(self._get('users/me'))
+
+    def get_users(self, inactive=False):
         """
         Get data on all users.
 
+        :param inactive: Gets inactive users instead of normal ones.
         :return: List of User objects.
         """
-        return [User(raw) for raw in self._get('users')['user']]
+        path = 'users' + ('/inactive' if inactive else '')
+        if 'total' not in self._get(path):
+            return []
+        return [User(raw) for raw in self._get(path)['user']]
 
-    def get_user(self, user_id):
+    def get_user(self, user_id, inactive=False):
         """
         Get data on an individual user.
 
         :param user_id: ID of user on whom to get data.
         :return: User object.
         """
-        return User(self._get('users/%s' % user_id))
+        return User(self._get(('users/' + ('inactive/' if inactive else '') + '%s') % user_id))
+
+    def create_user(self, user, user_id):
+        """
+        Create a new user.
+
+        :param user: User object containing necessary fields.
+        :param user_id: ID of user you wish to create.
+        :return: User object obtained from API.
+        """
+        return User(self._post('users/%s' % user_id, user.json()))
+
+    def create_users(self, users):
+        """
+        Bulk create users.
+
+        :param users: A list of users
+        :return: User objects obtained from API.
+        """
+        userList = [user.json() for user in users]
```

Comment:
Two things:
In Python, it's standard to use underscore_case instead of camelCase.

Also, I think you can go without the `userList` variable entirely.